src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,12 @@
+package duke.task;
+
+public class ToDo extends Task{
```

Comment:
Do take note of the spacing for each bracket and curly braces for the consistency of the whole program.